packages/vue/src/auth/entitlements.ts

```diff
@@ -59,8 +46,8 @@ const useEntitlementsQueryData = (customAttributes?: CustomAttributes) => {
 */
 export const useFeatureEntitlements = (key: string, customAttributes?: CustomAttributes): ComputedRef<Entitlement> => {
   return computed(() => {
-    const { entitlements, attributes, isV2 } = useEntitlementsQueryData(customAttributes);
-    return getFeatureEntitlements(entitlements, key, attributes, isV2)
+    const { entitlements, attributes } = useEntitlementsQueryData(customAttributes);
+    return getFeatureEntitlements(entitlements, key, attributes, true)
```

Comment:
dont forget to remove isV2 from `getFeatureEntitlements`

Reply:
will take more cycles to do it.. it's like a chicken and egg problem